### PR TITLE
Delete merge

### PR DESCRIPTION
**Pull request by Greg-Boggs**

### CODE REVIEW

**Comment by Greg-Boggs** [comment]
```
trigger ContactMergeTrigger on Contact (after delete) {
  List<Merged_Contact__c> listMergedContacts = new List<Merged_Contact__c>();
    for(Contact contact : trigger.old) {
        if(String.isNotBlank(contact.MasterRecordId)) { 
            listMergedContacts.add(new Merged_Contact__c(Contact__c = contact.MasterRecordId, Old_Contact__c = contact.id));  
        }         
    }
    if(listMergedContacts.size() > 0) {
        insert listMergedContacts;
    }    
}
```

**Comment by mshaver** [comment]
To get to the trigger action go here:
contact_triggers___salesforce_-_enterprise_edition

**Comment by Greg-Boggs** [comment]
screen shot 2016-09-30 at 10 33 48 am

**Comment by Greg-Boggs** [comment]
screen shot 2016-09-30 at 10 34 13 am

**Comment by mariacha** [comment]
Let's make sure to remember to update the API with the new hooks this code introduces.

File: salesforce.api.php [hunk 1]
```diff
@@ -372,5 +372,25 @@ function hook_salesforce_pull_entity_update($entity, $sf_object, $sf_mapping) {
...
+ *   The SF field name representing the original contact.
+ * @param string $merged_contact
+ *   The SF field name representing the merged contact.
+ * @param string $type
+ *   The SF object name representing the merge.
+ * @param string $modified_date
+ *   The SF field name representing last modified date of the contacts being
+ *   merged.
+ */
+function hook_salesforce_pull_entity_merge_fields_alter(&$old_contact, &$merged_contact, &$type, &$modified_date) {
```
Comment (mariacha) [inline salesforce.api.php]: @ruscoe your parameters are wrong here.

**Pull request merged**